lib/pf/UnifiedApi/Controller/Config.pm

```diff
@@ -1075,6 +1078,77 @@ sub fix_permissions {
     return $self->render(json => { message => $result });
 }
 
+
+sub audit_request {
+    my ($self) = @_;
+    my $status =  $self->res->code;
+    my $stash = $self->stash;
+    if (is_error($status) || !$stash->{auditable}) {
+        return;
+    }
+
+    my $record = $self->make_audit_record();
+    my $log = pf::dal::admin_api_audit_log->new($record);
+    $log->insert;
+}
+
+=head2 make_audit_record
+
+make_audit_record
+
+=cut
+
+sub make_audit_record {
+    my ($self) = @_;
+    my $stash = $self->stash;
+    my $status =  $self->res->code;
+    my $req = $self->req;
+    my $method = $req->method;
+    my $path = $req->url->path;
+    my $request = $self->make_audit_record_request();
+
+    my $current_user = $stash->{current_user};
+    return {
```

Comment:
You should also get the current tenant in this payload

Reply:
It is automatically added.